app/assets/javascripts/components/common/article_viewer_legend.jsx

```diff
@@ -25,23 +24,28 @@ const ArticleViewerLegend = ({ article, users, colors, status, allUsers, failure
 
       setUserLinks(users.map((user, i) => {
         let res;
+        // The 'unhighlightedContributors' array keeps track of the userids of users whose contributions
+        // were not successfully highlighted in the article viewer.
+        const isUserHovered = unhighlightedContributors.find(x => x === user.userid);
```

Comment:
It looks like this constant is actually keeping track of whether the user had any contributions attributed in the `unhighlightedContributors` array. I don't think `isUserHovered` is a good name for that.

Reply:
Okay will change variable name